db/table_cache.cc

```diff
@@ -64,8 +64,12 @@ void AppendVarint64(IterKey* key, uint64_t v) {
 }  // namespace
 
 TableCache::TableCache(const ImmutableCFOptions& ioptions,
+                       const MutableCFOptions& moptions,
                        const EnvOptions& env_options, Cache* const cache)
-    : ioptions_(ioptions), env_options_(env_options), cache_(cache) {
+    : ioptions_(ioptions),
+      moptions_(moptions),
```

Comment:
I don't see how the option change is propagated to here.

Reply:
right now through ColumnFamilyData's constructor `mutable_cf_options_(initial_cf_options_),` but I agree this doesn't seem to always have the latest mutablecfoptions.